### PR TITLE
[bridge] Add print registration info in cli

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

Add a subcommand to display current registration info.

## Test plan 

```
sui-bridge-cli print-sui-bridge-info --sui-rpc-url https://rpc.testnet.sui.io:443
```
---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

**Comment by dariorussi** [comment]
is this ready for review? it seems to be against a private branch and I am confused...

File: crates/sui-bridge-cli/src/lib.rs [hunk 1]
```diff
@@ -90,6 +90,12 @@ pub enum BridgeCommand {
         #[clap(long = "eth-rpc-url")]
         eth_rpc_url: String,
     },
+    /// Print current registration info
+    #[clap(name = "print-bridge-registration-info")]
+    PrintBridgeRegistrationInfo {
+        #[clap(long = "sui-rpc-url")]
```
Comment (dariorussi) [inline crates/sui-bridge-cli/src/lib.rs:96]: I know that you know that I know that you know... but you can just say `long` when the long name matches the field name. Though maybe was on purpose...

File: crates/sui-bridge-cli/src/main.rs [hunk 2]
```diff
@@ -187,6 +195,77 @@ async fn main() -> anyhow::Result<()> {
...
+                .into_iter()
+                .map(|summary| {
+                    let protocol_key =
+                        AuthorityPublicKeyBytes::from_bytes(&summary.protocol_pubkey_bytes)
+                            .unwrap();
+                    (summary.sui_address, (protocol_key, summary.name))
+                })
+                .collect::<HashMap<_, _>>();
+            let mut authorities = vec![];
+            for (_, member) in move_type_bridge_committee.member_registration {
```
Comment (dariorussi) [inline crates/sui-bridge-cli/src/main.rs:227]: you could work off references if you wanted particularly considering most of the usage is by reference. 
Just commenting to be honest, I do not think it matters much given the usage

**Review by dariorussi — APPROVED** [review]
all good a couple of comments just to show I read the PR

File: crates/sui-bridge-cli/src/main.rs (hunk 2 above)
Comment (longbowlu) [reply crates/sui-bridge-cli/src/main.rs:227]: yeah we consume it anyway here

File: crates/sui-bridge-cli/src/lib.rs (hunk 1 above)
Comment (longbowlu) [reply crates/sui-bridge-cli/src/lib.rs:96]: will fix this in a later PR

**Pull request merged**